corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/util/util.js

```diff
@@ -0,0 +1,24 @@
+function Util() {
+}
+
+Util.getQueryParams = function (qs) {
+    qs = qs.split("+").join(" ");
+
+    var params = [], tokens,
+        re = /[?&]?([^=]+)=([^&]*)/g;
+
+    while (tokens = re.exec(qs)) {
```

Comment:
You can also disable this rule by adding
```
"no-cond-assign": ["off"],
```
To the [rules config](https://github.com/dimagi/commcare-hq/blob/master/.eslintrc.js#L26) if you feel strongly about it.